### PR TITLE
Remove elrond-go and arwen dependencies

### PR DESCRIPTION
**Pull request by miiu96**
- Remove elrond-go and arwen dependencies

### CODE REVIEW

File: common_test.go [hunk 1]
```diff
@@ -11,24 +11,23 @@ import (
 	"github.com/ElrondNetwork/elastic-indexer-go/data"
 	"github.com/ElrondNetwork/elastic-indexer-go/mock"
 	"github.com/ElrondNetwork/elrond-go-core/core"
+	nodeData "github.com/ElrondNetwork/elrond-go-core/data"
```
Comment (bogdan-rosianu) [inline common_test.go:14]: coreData?

File: data/account.go [hunk 2]
```diff
@@ -3,7 +3,7 @@ package data
 import (
 	"time"
 
-	"github.com/ElrondNetwork/elrond-go/state"
+	nodeData "github.com/ElrondNetwork/elrond-go-core/data"
```
Comment (bogdan-rosianu) [inline data/account.go:6]: coreData

File: processTransactions.go [hunk 3]
```diff
@@ -162,8 +154,12 @@ func (tdp *txDatabaseProcessor) addScrsReceiverToAlteredAccounts(
 	}
 }
 
+// RefundGasMessage is the message returned in the data field of a receipt,
+// for move balance transactions that provide more gas than needed
+const RefundGasMessage = "refundedGas"
```
Comment (bogdan-rosianu) [inline processTransactions.go:159]: move it to data package?

File: workItems/workItemAccounts.go [hunk 4]
```diff
@@ -2,20 +2,20 @@ package workItems
 
 import (
 	"github.com/ElrondNetwork/elastic-indexer-go/data"
-	"github.com/ElrondNetwork/elrond-go/state"
+	nodeData "github.com/ElrondNetwork/elrond-go-core/data"
```
Comment (bogdan-rosianu) [inline workItems/workItemAccounts.go:5]: coreData

File: workItems/workItemAccounts_test.go [hunk 5]
```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/ElrondNetwork/elastic-indexer-go/data"
 	"github.com/ElrondNetwork/elastic-indexer-go/mock"
 	"github.com/ElrondNetwork/elastic-indexer-go/workItems"
-	"github.com/ElrondNetwork/elrond-go/state"
+	nodeData "github.com/ElrondNetwork/elrond-go-core/data"
```
Comment (bogdan-rosianu) [inline workItems/workItemAccounts_test.go:10]: coreData

File: dataIndexer.go [hunk 6]
```diff
@@ -124,16 +122,11 @@ func (di *dataIndexer) SaveValidatorsPubKeys(validatorsPubKeys map[uint32][][]by
 }
 
 // SaveAccounts will save the provided accounts
-func (di *dataIndexer) SaveAccounts(timestamp uint64, accounts []state.UserAccountHandler) {
+func (di *dataIndexer) SaveAccounts(timestamp uint64, accounts []nodeData.UserAccountHandler) {
```
Comment (bogdan-rosianu) [inline dataIndexer.go:125]: coreData

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

**Review by iulianpascalau — COMMENTED** [review]
only what Bogdan found

File: common_test.go (hunk 1 above)
Comment (miiu96) [reply common_test.go:14]: done.

File: data/account.go (hunk 2 above)
Comment (miiu96) [reply data/account.go:6]: done.

File: processTransactions.go (hunk 3 above)
Comment (miiu96) [reply processTransactions.go:159]: done.

File: workItems/workItemAccounts.go (hunk 4 above)
Comment (miiu96) [reply workItems/workItemAccounts.go:5]: done.

File: workItems/workItemAccounts_test.go (hunk 5 above)
Comment (miiu96) [reply workItems/workItemAccounts_test.go:10]: done.

File: dataIndexer.go (hunk 6 above)
Comment (miiu96) [reply dataIndexer.go:125]: done.

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**